packages/react-positioning/src/utils/getBoundary.ts

```diff
@@ -12,7 +12,7 @@ export function getBoundary(element: HTMLElement | null, boundary?: Boundary): P
   }
 
   if (boundary === 'scrollParent') {
-    let boundariesNode = getScrollParent(element);
+    let boundariesNode: HTMLElement | undefined = getScrollParent(element);
```

Comment:
ah good catch, in that case can you remove the `as` cast in the return on line 18 ? 

```typescript
return boundariesNode
```

Reply:
Typecast was removed